### ISSUE TITLE
Below the headline a line is shown 

### ISSUE THREAD

**Issue by stefan-franz** [danielroehrig/diary#67]
In the view mode a line below a headline is shown - i think it's a bug.

**Comment by danielroehrig** [comment]
Can you attach a screenshot please?

**Comment by stefan-franz** [comment]
![Line below the headline](https://github.com/danielroehrig/diary/assets/96744684/ec1c8fa9-02e4-4b5f-a23e-3cf5d8276e9b)

**Comment by danielroehrig** [comment]
I think that is intentional. I wanted to have a separation between the headline and the content. But I see how that could be confusing. I will keep it in mind.

**Comment by stefan-franz** [comment]
I think it's not necessary. Normal .md files don't have it.

**Comment by danielroehrig** [comment]
Yeah, I liked the visual separation. But I complained a lot about having to do the frontend part, so if you want the line gone and maybe improve anything else, I'd more than happy to accept your PR. :smile: 

Otherwise, I will do it whenever I find the time. I think you are right about this.

**Comment by stefan-franz** [comment]
I don't be a programmer - sorry. I think for an pro like you it's in 5min finished. 
Greetings, Stefan
